### DISCUSSION TITLE
Third person access

### DISCUSSION THREAD

**Discussion by gaijin-developer** [vercel/community#6707] [Feature Question]
### Question

I would like to give access to a client to view a project but i want him to have access to the analytics of the deployment but only that Deployment and nothing else. If i give someone access to a deployment, can they see the analytics of the deployment? and will they be able to see the other deployments  or only the one i add them to ?

**Comment by versecafe** [comment]
@gaijin-developer if you give access to a deployment someone can only view the deployment in order to share the analytics you would need to either add them to the Vercel team or parse out your analytics data with a tool like [VAAS](vaas-brown.vercel.app) or it's underlying functions to get the data in json, csv, or yaml to share with your client.
